src/components/BaseModal/BaseModal.styled.jsx

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const ModalBackground = styled.div`
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
```

Comment:
its better to use 
```
top: 0;
bottom: 0;
left: 0;
right: 0
```

100% and 100vh don't work predictable on some devices, and some scroll could appear or vice versa - some empty space.